op-node/node/config.go

```diff
@@ -34,17 +38,38 @@ type RPCConfig struct {
 	ListenPort int
 }
 
+type MetricsConfig struct {
+	Enabled    bool
+	ListenAddr string
+	ListenPort int
+}
+
+func (m MetricsConfig) Check() error {
+	if !m.Enabled {
+		return nil
+	}
+
+	if m.ListenPort < 0 || m.ListenPort > math.MaxUint16 {
```

Comment:
Could maybe type it as `uint16` directly, if the CLI library supports `uint16`. This works too.

Reply:
It doesn't, unfortunately :(. Just `uint` and `uint64`.